quoting-service/templates/config.yaml

```diff
@@ -12,7 +12,7 @@ data:
   default.json: {{ (tpl (.Files.Get "configs/default.json") . ) | quote }}
   {{- if .Values.rules }}
   {{- if (typeIs "string" .Values.rules.template) }}
-  rules.json: {{ (tpl .Values.rules.template . ) }}
+  rules.json: {{ (tpl .Values.rules.template . ) | quote }}
```

Comment:
I _think_ this might be better
```suggestion
  rules.json: |
{{ (tpl .Values.rules.template . ) | indent 4 }}
```

Reply:
You can even use `>` in this case since it's one line

Reply:
Doesn't work, just tried.

Reply:
![image](https://user-images.githubusercontent.com/2804336/90004655-e644f600-dc8d-11ea-8a2b-4c35f643fcb3.png)

Reply:
Yes I guess it can be. The advantage of `|` in this case is that the file that gets mounted in the container will retain all of the nice formatting. But if `>` works out even slightly easier to implement it probably wins.

Reply:
`quote` works elsewhere in this file, it appears to escape text appropriately.